src/tracking.c

```diff
@@ -422,12 +414,10 @@ void trackingHandlePendingKeyInvalidations() {
 
     listRewind(server.tracking_pending_keys,&li);
     while ((ln = listNext(&li)) != NULL) {
-        invalidatedKey *ik = listNodeValue(ln);
-        client *target = lookupClientByID(ik->client_id);
-        if (target != NULL)
-            sendTrackingMessage(target,(char *)ik->key->ptr,sdslen(ik->key->ptr),0);
-        decrRefCount(ik->key);
-        zfree(ik);
+        robj *key = listNodeValue(ln);
+        if (server.current_client != NULL)
```

Comment:
so if current_client is null we don't send anything?
i'd drop that `if` and even add an assertion with some comment explaining it.

Reply:
I'm thinking whether it is possible that some commands put entries in the list, then client is freed.

Reply:
ok, makes sense, that means we can't put the assertion.
let's at least add a comment explaining that `if`